tests/test_registry.py

```diff
@@ -52,10 +53,14 @@ async def test_basic_certificate_validator_tls_aia(domain):
     # southwest.com -> application/pkcs7-mime
     # xuite.net     -> application/x-pkcs7-certificates
     # icpedu.rnp.br -> binary/octet-stream (PEM, PKCS#7)
+    # repositorio.serpro.gov.br -> binary/octet-stream (DER, PKCS#7)
 
     icpedu_root = load_cert_object('testing-aia', 'root-icpedu.rnp.br')
+    serpro_root = load_cert_object(
+        'testing-aia', 'root-repositorio.serpro.gov.br'
+    )
     trust_manager = SimpleTrustManager.build(
-        extra_trust_roots=[icpedu_root],
```

Comment:
Done! I added a new file `test_common_utils.py` with a unit test covering the problem that I was having when unpacking a pkcs7 with the wrong alias returned from the server. 
I kind of added the test that you asking for, instead of mocking `CertificateFetcher` I mocked `RequestCertificateFetcher` and mocked the its `fetch_certs` function. I did some tests mocking the `CertificateFetcher` as you asked, but for some reason the original code  it just started working when I was just returning a list of certs (I think there's some strange behavior happening with all the asynchronous calls that I couldn't replicate it).